Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserTreeView.cpp

```diff
@@ -573,6 +578,110 @@ namespace AzToolsFramework
                 QFile::copy(oldPath.c_str(), newPath.c_str());
             }
         }
+
+        void AssetBrowserTreeView::MoveEntries()
+        {
+            using namespace AzFramework::AssetSystem;
+            EntryTypeFilter* foldersFilter = new EntryTypeFilter();
+            foldersFilter->SetEntryType(AssetBrowserEntry::AssetEntryType::Folder);
+
+            auto selection = AzToolsFramework::AssetBrowser::AssetSelectionModel::EverythingSelection();
+            selection.SetTitle("folder to move to");
```

Comment:
If this is user-facing could be something like this instead:

```suggestion
            selection.SetTitle(tr("Folder to move to"));
```

Reply:
This isn't needed as the AssetPickerDialog uses this:-

Reply:
`tr` only operates on string literals, so it won't translate that string dynamically. The literal string `Folder to move to` does need its own `tr`.